docs/howtoguides/enable_fips.rst

```diff
@@ -25,24 +26,32 @@ To enable FIPS, run:
 
 .. code-block:: bash
 
-    $ sudo pro enable fips
+    $ sudo pro enable fips-updates
 
 You should see output like the following, indicating that the FIPS packages
 have been installed:
 
 .. code-block:: text
 
-    Installing FIPS packages
-    FIPS enabled
+    Installing FIPS Updates packages
+    FIPS Updates enabled
     A reboot is required to complete install.
 
 Enabling FIPS should be performed during a system maintenance window since
 this operation makes changes to underlying SSL-related libraries and requires
 a reboot into the FIPS-certified kernel.
 
+.. note::
+
+    ``fips-updates`` is the recommended way of enabling FIPS, as it will bring
+    the FIPS-compliant packages with security updates and coverage. There is
+    also a ``fips`` service, which has only the certified packages, *without
+    further security updates*, but that is deprecated from Ubuntu 22.04 (Jammy
+    Jellyfish) onward.
+
```

Comment:
The content above this note is really confusing, because there is a difference between fips and fips-updates, and we're installing but referring to the other. 

I think this note needs to be the introduction to the page as body text, and not a note, because that makes it clearer why there is this confusion.

Better would be to move the note (as mentioned) but also change the references to FIPS throughout the document to "FIPS with updates" or however we want to call it.

Reply:
so this is a product decision. From Jammy onwards, `fips` does not even exist, and the concept of "enabling FIPS" is achieved using the `fips-updates` service. We don't want people enabling `fips`. @henrycoggillcnc can explain it better if needed